README.md

```diff
@@ -28,25 +27,26 @@ Requirements:
 * ninja
 * Java: Any JDK8 or above, ensure JAVA_HOME is set
 * Maven
-1) brew install maven cmake3 ninja (if you have homebrew installed, otherwise install these manually)
+1) brew install maven cmake3 (if you have homebrew installed, otherwise install these manually)
 2) git clone https://github.com/awslabs/aws-crt-java.git
 3) cd aws-crt-java
 4) mvn compile
 
 ### Windows
 Requirements:
 * Visual Studio 2015 or above
-* CMake 3
+* CMake 3.1
 * Java: Any JDK8 or above, ensure JAVA_HOME is set
 * Maven
-1) choco install maven (if you have chocolatey installed), otherwise install maven manually
+1) choco install maven (if you have chocolatey installed), otherwise install maven and the JDK manually
 2) git clone https://github.com/awslabs/aws-crt-java.git
 3) cd aws-crt-java
 4) mvn compile
 
 ## Installing
 From the aws-crt-java directory:
-```mvn install:install-file -Dfile=target/aws-crt-java-1.0.jar -DpomFile=pom.xml```
+```mvn install```
```

Comment:
This runs the tests, which fail without all the parameters for certs and the like.